sdks/java/extensions/sql/build.gradle

```diff
@@ -149,15 +149,13 @@ task runPojoExample(type: JavaExec) {
 task integrationTest(type: Test) {
   group = "Verification"
   def gcpProject = project.findProperty('gcpProject') ?: 'apache-beam-testing'
-  def gcpRegion = project.findProperty('gcpRegion') ?: 'us-central1'
   def gcsTempRoot = project.findProperty('gcsTempRoot') ?: 'gs://temp-storage-for-end-to-end-tests/'
 
   // Disable Gradle cache (it should not be used because the IT's won't run).
   outputs.upToDateWhen { false }
 
   def pipelineOptions = [
           "--project=${gcpProject}",
```

Comment:
If they don't run on Dataflow, what is project and other gcp flags are used for?

Reply:
These tests use other GCP resources like BigQuery and Pub/Sub.